uploading-data.md

```diff
@@ -8,12 +8,47 @@ has_children: true
 
 # Uploading Data
 
-Centers do not need to follow a particular folder hierarchy in the provided cloud storage location.
+Centers may use a hierarchical file structure within their cloud storage location. However, each root (top-level) folder and all of its subfolders **must** contain data of the same type (see details below). This will help separate datasets and make annotating metadata with the data curator app easier for users. 
 
-To upload data to your DCC-designated storage location, please use the Synapse platform tools.
+## Top Level Folder Names
+Top level folders correspond to the assay and level metadata you will be submitting. For example we have scRNAseq levels 1, 2, 3, 4 and in each RFC there is a corresponding attribute table that is converted into a schema and google spreadsheet for the Data Curator App. This means that there would be top level folders `single_cell_RNAseq_level_1`, `single_cell_RNAseq_level_1`, `single_cell_RNAseq_level_3`, and `single_cell_RNAseq_level_4`
+
+```
+ .
+├── clinical_tier_1_diagnosis
+├── clinical_tier_1_demographics
+├── biospecimen_tier_1
+│   ├── file1.txt
+│   └── file2.txt
+└── single_cell_RNAseq_level_1
+    ├── fileA.fq
+    └── fileB.fq
+
+```
+Your files should be reasonably descriptive in stating the assay type and level and in consistently prefixed with the assay type because they will appear on the Data Curator App in alphanumeric order. e.g. please preface all your clinical folders `clinical_tier_[n]_` so they will show up together instead of `diagnosis_clinical_[n]` and `demographics_clinical_[n]`
+
+## Example Folder Hierarchy (optional)
+
+Subfolders **must** be of the same data type and level as the root folder they are contained in. For example you can _not_ put a biospecimen Tier 1 and clinical demographics Tier 1 subfolder within the same folder. 
+```
+.
+├── clinical_tier_1_diagnosis
+├── clinical_tier_1_demographics
+├── biospecimen_tier_1
+│   ├── experiment_1
```

Comment:
Another option would be to have each batch experiment as a top-level folder; e.g.
- biospecimen_tier_1_experiment_1
- biospecimen_tier_1_experiment_2
- single_cell_level_1_batch_1
- single_cell_level_1_batch_2

In the hierarchy case they would fill in one manifest including all files in experiment/batches; in the "flattened" case they would fill in one manifest for each top level folder.

Not sure if we want to document these options and the differences wrt manifest completion? @xdoan @clarisse-lau  @ychae - thoughts?

Reply:
I think this would be a good option. 

For the `Top Level Folder Names` can we add that we will seed the "basic" top level folders in buckets. Then we can add a paragraph for Milen's case such as : 

"Additionally, your top level folder names can explicitly refer to a batch or experiment, e.g. (insert example). Note that this might create many top level folders, see section below about subfolders."

Reply:
@xdoan and @milen-sage I've added in your suggestions -- please let me know if I integrated them correctly!